### DISCUSSION TITLE
Global- and category-level filters

### DISCUSSION THREAD

**Discussion by upstreamist** [FreshRSS/FreshRSS#3608] [Ideas]
Maybe this is already possible, though I haven't seen it documented anywhere. I'd like to be able to apply filters to entire categories, or even to every feed on the instance. That way, if there's a subject I'd like to filter out of multiple feeds, I can apply it to the category, and have the one filter apply to all of the feeds in that category — or site-wide, if I apply it globally.

Preferably, filters would have higher precedence the fewer feeds they affect. In other words, if I have a filter applied (say, `inurl:foo`) to an entire category, and I want to exempt a single feed in that category, I can reverse it at the feed level (with `-inurl:foo` or `!inurl:foo`).

**Comment by Alkarex** [comment]
It is not yet possible, but will come soonish, after this PR https://github.com/FreshRSS/FreshRSS/pull/3303#issuecomment-752644798
